### ISSUE TITLE
Configuration Error: Included "outputs" are off by 1

### ISSUE THREAD

**Issue by jshank** [BioSehnsucht/ha-elkm1#9]
Referring to [gwww branch](/BioSehnsucht/ha-elkm1/tree/gwww-elkm1-lib) 
In `configuration.yaml`

```
elkm1:
  host: elk://192.168.0.1:2101 
  output:
    include:
      - 2

```
renders `switch.elkm1_output_003` available (one output greater the included output)

**Comment by BioSehnsucht** [comment]
Including a single item seems to be off by one, but a range appears to work as expected.

**Comment by BioSehnsucht** [comment]
@jshank Fixed in latest commit, but last few commits are based on gwww version not yet on PyPi, so if you don't manually update Gwww's lib you may have issues.

**Comment by jshank** [comment]
Confirmed fixed in 48d41c5a18c4966a406588ab2aaffdff9fbb995e. Thanks!
